parts/linux/cloud-init/artifacts/cse_main.sh

```diff
@@ -100,6 +100,17 @@ logs_to_events "AKS.CSE.disableSystemdResolved" disableSystemdResolved
 
 logs_to_events "AKS.CSE.configureAdminUser" configureAdminUser
 
+
+if [[ "${SHOULD_CONFIG_MEMLOCK}" == "true" ]]; then
+  # TODO - add error handling
+  echo "* hard memlock ${MEM_LOCK_VAL}" >> /etc/security/limits.conf || exit $ERR_UPDATE_CA_CERTS
```

Comment:
do you know what the default content of this file is? particularly on mariner @hbeberman?

you can find out in e2e, but wondering if simply append makes sense vs a drop in for /etc/security/limits.d  as a seperate file with our constructions?

does dupe later in same file take higher precedence? or error?

Reply:
can we move the timing of this closer to sysctls with other custom node config, but before kubelet start?